db/migrate/20171121120455_rebuild_classification_indexes.rb

```diff
@@ -0,0 +1,57 @@
+class RebuildClassificationIndexes < ActiveRecord::Migration
+  disable_ddl_transaction!
+  TABLE_NAME = :classifications
+
+  def change
+    reversible do |direction|
```

Comment:
TIL about this.

Reply:
this was new for me too :) quite useful for the index builds